src/programs/symmetry_expand_stack_and_par/symmetry_expand_stack_and_par.cpp

```diff
@@ -537,14 +540,22 @@ bool SymmetryExpandStackAndPar::DoCalculation()
 
 	ProgressBar *my_progress;
 
+	if (do_centring_and_cropping == true)
+	{
+		cropped_image.Allocate(cropped_box_size, cropped_box_size, 1);
+		original_x_coord = centre_x_coord - input_stack.ReturnXSize() / 2;
```

Comment:
Looks to me like `original_x_coord` and y and z will get overwritten around line 657 - is this intentional? 
Also on those lines, you'd used `physical_address_of_box_center`, not `XSize() / 2` - not sure that's relevant

Reply:
This is intentional, when not doing subtraction their is no volume to get the dimensions from,  above i am assuming it is a cube the same as the particle stack which should be fine in most cases, but if doing subtraction I overwrite with the actual volume info.

Reply:
Fair enough!